src/ostorlab/runtimes/local/models/models.py

```diff
@@ -595,10 +596,43 @@ def create_from_agent_group_definition(
                 description=agent_group_definition.description,
                 agents=agents,
             )
+
+            ag_asset_types = []
+            for asset_type in asset_types:
+                ag_asset_types.append(AssetType.create(type=asset_type))
+
+            agent_group.asset_types = ag_asset_types
             session.add(agent_group)
             session.commit()
             return agent_group
 
+    @staticmethod
+    def create_from_files(agent_groups_path: str) -> list["AgentGroup"]:
+        """Create agent groups from a file.
+
+        Args:
+            agent_groups_path: Path to the agent groups folder.
+
+        Returns:
+            List of agent groups.
+        """
+
+        agent_groups = []
+        agent_groups_path = pathlib.Path(agent_groups_path)
+        for agent_group_file in agent_groups_path.iterdir():
+            if agent_group_file.is_file() and agent_group_file.suffix == ".yaml":
+                with open(agent_group_file, "r") as file:
+                    agent_group_definition = definitions.AgentGroupDefinition.from_yaml(
+                        file
+                    )
+                    asset_type = agent_group_file.stem
+                    agent_group = AgentGroup.create_from_agent_group_definition(
```

Comment:
Does this check if the agent group exists or it always creates?

Reply:
for now they are always created, to check